app/src/main/java/vn/mbm/phimp/me/PhimpMe.java

```diff
@@ -63,7 +67,8 @@
 //        forceCloseDialogAfterToast = false,
 //        resToastText = R.string.crash_report_text)
 @SuppressWarnings("deprecation")
-public class PhimpMe extends AppCompatActivity implements BottomNavigationView.OnNavigationItemSelectedListener //, android.view.GestureDetector.OnGestureListener
+public class PhimpMe extends AppCompatActivity implements BottomNavigationView.OnNavigationItemSelectedListener,
+        ViewPager.OnPageChangeListener //, android.view.GestureDetector.OnGestureListener
```

Comment:
@vinaysajjanapu is this the code for adding swipe gesture? These changes were already reverted. Rebase your code to current development branch.
Make sure it show only the relevant additions.